extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -29,6 +30,9 @@ import {renderAsElement} from './simple-template';
 /** @struct @typedef {{className: string, triggers: (string|undefined)}} */
 let ButtonState_1_0_Def; // eslint-disable-line google-camelcase/google-camelcase
 
+/** @private {?../../../src/service/viewer-interface.ViewerInterface} */
+this.viewer_ = null;
```

Comment:
You're not within an object so I assume the `this` refers to the global scope here? Please remove and use a local variable line 342.